backend/migrations/20241026085259_initial_scheme.sql

```diff
@@ -2,15 +2,42 @@ create type membership_status as enum ('pending', 'member', 'extraordinary', 'no
 
 create table "user"
 (
-    id         uuid primary key,
-    first_name text              not null,
-    last_name  text              not null,
-    roles      jsonb             not null,
-    status     membership_status not null,
-    email      text              not null unique,
-    pw_hash    text,
-    created    timestamptz       not null,
-    updated    timestamptz       not null
+    id                uuid primary key,
+    first_name        text              not null,
+    last_name         text              not null,
+    phone             text              not null,
+    student_number    bigint,
+    nkbv_number       bigint,
+    sportcard_number  bigint,
+    ice_contact_name  text              not null,
+    ice_contact_email text              not null,
+    ice_contact_phone text              not null,
```

Comment:
Would be nice to enforce ˋnot nullˋ in the DB, but I think we can't if not all current users habe ICE details provided, as we cannot migrate the DB then